src/utils/sources-tree/utils.js

```diff
@@ -34,7 +34,7 @@ export function isInvalidUrl(url: Object, source: Object) {
   return (
     IGNORED_URLS.indexOf(url) != -1 ||
     !source.get("url") ||
-    source.get("loading") ||
+    source.loadedState === "loading" ||
```

Comment:
I tried using the isLoaded method for the above condition but it was throwing some error and the sources didn't get loaded during initial load. So checked for the value manually.